docs/source/scripts/onnx/build_onnx_supported_aten_op_csv_table.py

```diff
@@ -8,18 +8,59 @@
 from torch.onnx import _onnx_supported_ops
 
 # Constants
-BUILD_DIR = "build"
-AUTO_GEN_ATEN_OPS_CSV_FILE = "auto_gen_aten_op_list.csv"
+BUILD_DIR = "build/onnx"
+SUPPORTED_OPS_CSV_FILE = "auto_gen_supported_op_list.csv"
+UNSUPPORTED_OPS_CSV_FILE = "auto_gen_unsupported_op_list.csv"
+
+
+def _sort_key(namespaced_opname):
+    return tuple(reversed(namespaced_opname.split("::")))
+
+
+def _get_op_lists():
+    all_schemas = _onnx_supported_ops.all_forward_schemas()
+    symbolic_schemas = _onnx_supported_ops.all_symbolics_schemas()
+    supported_result = set()
+    not_supported_result = set()
+    for opname in all_schemas:
+        if opname.endswith("_"):
+            opname = opname[:-1]
+        if opname in symbolic_schemas:
+            # Supported op
+            opsets = symbolic_schemas[opname].opsets
+            supported_result.add(
+                (
+                    opname,
+                    f"Since opset {opsets[0]}",
+                )
+            )
+        else:
+            # Unsupported op
+            not_supported_result.add(
+                (
+                    opname,
+                    "Not yet supported",
```

Comment:
What do you think if we add a future support version number here, something like: "Col2Im is not yet supported, but will be in version 18" => or these include in supported already? If that's the case, we should think about "symbolic fn supported, but onnx version hasn't caught up"?

Reply:
That is going to be helpful! Is there a good way to obtain this information? If a symbolic is defined, we count it as supported

Reply:
A stupid one that I can think of is to retrieve the info from `onnx_backend_test.py` as we usually disable new op there for caffe2. Otherwise, parse the supported Op from onnx folder 🤔